regression-test/suites/external_table_p0/tvf/test_queries_tvf.groovy

```diff
@@ -0,0 +1,22 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_queries_tvf","p0,external,tvf,external_docker") {
+    def res = sql """ select * from queries(); """
```

Comment:
better test it more precise. eg, you can first send a query to with a uniq id(such as where id=uuid).
And then select from queries() to see if the certain query exist.

Reply:
done